app/AppState/types/TransactionType.ts

```diff
@@ -1,7 +1,8 @@
+import { TransactionTypeEnum } from '../enums/TransactionTypeEnum';
 import TxDetailType from './TxDetailType';
 
 export default interface TransactionType {
-  type: 'Sent' | 'Received' | 'SendToSelf'; // like kind
+  type: TransactionTypeEnum.Sent | TransactionTypeEnum.Received | TransactionTypeEnum.SendToSelf; // like kind
```

Comment:
Here is an exception, because the Enum have 4 items (I receive from `zingolib` transactions with those different types) but the App only use three of them (The `Fee` is included as a part/field of the transactions). Because of that I use here only three items.